readme.md

```diff
@@ -7,67 +7,257 @@
 
 # Point One Navigation Polaris Service #
 
-This repo provides a library for interacting with the Point One Navigation Polaris Service to receive GNSS RTK corrections. Compatible GNSS receivers can acheive better than 10cm accuracy when used with the service. Supported receivers are currently available from Septentrio and u-Blox. We also offer a hardware SDK you can request by emailing [info@pointonenav.com](mailto:info@pointonenav.com)
+This repo provides C and C++ libraries for interacting with the Point One Navigation Polaris Service to receive GNSS RTK
+corrections. Compatible GNSS receivers can achieve better than 10 cm accuracy when used with the service.
```

Comment:
Put the hardware devkit line back, and maybe instead of email, we should link to https://pointonenav.com/devkits

Reply:
oh disregard, you have tihs below.